test/parallel/test-repl-async-iife.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+require('../common');
+
+// Note: This test ensures that async IIFE doesn't crash
```

Comment:
Do you know the Node.js version for which this test would crash? It only affects debug builds, so it would probably require building Node.js locally for confirming that this is actually a regression test for the issue.

Seeing https://github.com/nodejs/node/issues/38685#issuecomment-976388169 it appears that it got fixed later on, so I'm curious what the fix was.

Reply:
Hey @RaisinTen 👋 

Honestly I don't have a clue on this :/ 

Maybe @addaleax could provide the version.

Reply:
it affected the `master` branch in May 2021, so I would try with with v14.0.0.

Reply:
Thanks @targos :)

Should I checkout the v14 branch and run the test it and see if it fails?